src/vl-radio-group.js

```diff
@@ -19,14 +19,31 @@ export class VlRadioGroup extends vlElement(HTMLElement) {
   }
 
   connectedCallback() {
-    this.radios.forEach((radio) => {
-      radio.setAttribute('data-vl-name', 'radio');
-    });
+    this._groupRadios();
+    this._transmitFocus();
   }
 
   get radios() {
     return this.querySelectorAll('vl-radio');
   }
+
+  _groupRadios() {
+    this.radios.forEach((radio) => radio.setAttribute('data-vl-name', 'radio'));
+  }
+
+  _transmitFocus() {
+    this.tabIndex = 0;
+    this.addEventListener('focus', () => {
+      this.addEventListener('keyup', (event) => {
+        if (event.shiftKey) {
+          this.radios[this.radios.length - 1].focus();
```

Comment:
is er geen risico om out of bounds te gaan? of gaan we ervan dat er altijd radio's in de radiogroup zitten?

Reply:
Een radio-group zonder radio's lijkt me niet interessant, maar het is veiliger om ervoor te zorgen dat er ook dan niet out of bounds gegaan wordt. Ik zal de code uitbreiden en een test toevoegen.